h2o-core/src/main/java/water/MRTask.java

```diff
@@ -386,14 +386,58 @@ private StringBuilder print(StringBuilder sb, int d) {
   public final T doAll( Vec vec, boolean run_local ) { return doAll(null,vec, run_local); }
   public final T doAll(byte[] types, Vec vec, boolean run_local ) { return doAll(types,new Frame(vec), run_local); }
 
-  /** Invokes the map/reduce computation over the given Frame.  This call is
-   *  blocking.  */
+  /** Invokes the map/reduce computation over the given Frame. This call is
+   *  blocking.  
+   */
   public final T doAll( Frame fr, boolean run_local) { return doAll(null,fr, run_local); }
+  
+  /** Invokes the map/reduce computation over the given Frame. This call is
+   *  blocking. The run is performed globally.
+   */
   public final T doAll( Frame fr ) { return doAll(null,fr, false); }
```

Comment:
missing @param docs since you are adding those to the methods below

Reply:
I will add them here and on more places when I am sure the wording is right.

Reply:
fixed